src/selectors_vm/ast.rs

```diff
@@ -49,15 +49,15 @@ impl NthChild {
 pub(crate) enum OnTagNameExpr {
     ExplicitAny,
     Unmatchable,
-    LocalName(String),
```

Comment:
Why are we replacing `Strings` with `Box<str>`?

Reply:
I've noticed that these strings aren't grown anywhere, and are always created from `&str`, so we're carrying a redundant capacity for no reason.

Newer versions of `selectors` require storing a precomputed hash, so this makes room for adding it without regressing struct sizes.

Reply:
Sounds good.